finesse/hardware/__init__.py

```diff
@@ -1,8 +1,14 @@
 """This module contains code for interfacing with different hardware devices."""
+import sys
+
 from pubsub import pub
 
+if "--dummy-em27" in sys.argv:
```

Comment:
Do we want equivalent statements for all other dummy devices?

Reply:
Yes, probably. We want some way to be able to switch between using dummy devices or real ones and probably the easiest way to do that is with command-line arguments or environment variables.

Atm you *only* have the option to have a dummy stepper motor though -- even though I've written the code for the real one. My plan is to make the GUI controls for connecting to the serial devices actually do something and then we can integrate it into the main program.